components/Button/TransactionButton.tsx

```diff
@@ -78,10 +78,12 @@ export function TransactionButton({
         message = 'Failed';
         break;
     }
-    const transactionLink = (
-      <EtherscanTransactionLink transactionHash={transactionData!.hash}>
+    const transactionLink = transactionData ? (
+      <EtherscanTransactionLink transactionHash={transactionData.hash}>
         view transaction
       </EtherscanTransactionLink>
+    ) : (
+      ''
```

Comment:
ran into a runtime error the first time I did this because `transactionData` didn't exist. Handling that case.